app/packages/looker-3d/src/labels/line.tsx

```diff
@@ -1,6 +1,23 @@
 import React from "react";
 import * as THREE from "three";
 import { OverlayProps } from "./shared";
+import { useRecoilState } from "recoil";
+import { polylineLabelLineWidthAtom } from "../state";
+import { Line2 } from "three/examples/jsm/lines/Line2";
+import { LineMaterial } from "three/examples/jsm/lines/LineMaterial";
+import { LineGeometry } from "three/examples/jsm/lines/LineGeometry";
+import { extend, ReactThreeFiber } from "@react-three/fiber";
+
+extend({ Line2, LineMaterial, LineGeometry });
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      line2: ReactThreeFiber.Node<Line2, typeof Line2>;
+      lineGeometry: ReactThreeFiber.Node<LineGeometry, typeof LineGeometry>;
+    }
+  }
+}
```

Comment:
can we put these declaration merging / global augmentation code in a separate file? maybe `three-d-labels.d.ts`, or even better `index.d.ts` in looker-3d package root?